esp/scm/ws_dfu.ecm

```diff
@@ -812,7 +821,43 @@ ESPresponse [exceptions_inline, nil_remove] EraseHistoryResponse
     ESParray<ESPStruct History, Origin> History;
 };
 
+ESPStruct DFUPartLocations
+{
+    int LocationIndex;
+    string Location;
+};
+
+ESPStruct DFUPartCopies
+{
+    int PartId;
+    ESParray<string> LocationIndexes;
+};
+
+ESPrequest DFUReadAccessRequest
+{
+    string Name;
+    string Cluster;
+    int ExpirySeconds(60);
+    ESPenum SecAccessType AccessType;
+    string JobId;
+    bool ReturnFileInfo(false);
```

Comment:
@rpastrana - I've defaulted this to false. Which is good for the engines (at least for now), since they get a IDistributeFile with all meta before getting the security meta.
It also looks more symmetrical with the other optionals below.

Reply:
Reasonable.